balena/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+echo "TASKINFO: Build the application in BalenaCloud"
+set -e
+SRC_PATH="$1"
+set -u
+
+test "${DEBUG}" == "false" || set -x
+
+balena_app_exists() {
+  return $(balena apps | grep -q "$1")
+}
+
+get_project_name_from_source() {
+    echo "$(git -C $SRC_PATH remote -v | head -1 | sed 's|.*/\(.*\)\.git.*$|\1|g')"
+}
+
+app_name=$(get_project_name_from_source)
+
+balena login -t $API_KEY
```

Comment:
Will we share the same API key with all github orgs for now?

Reply:
Yes, at this moment we are using an API_KEY generated for the balenalabs user. It is read from the Vault and injected into the script as as an ENV parameter from the job definition